content/apps/using-oauth2.md

```diff
@@ -47,7 +47,7 @@ https://my-pim.cloud.akeneo.com/connect/apps/v1/authorize?
 ```
 
 ::: info
-You can consult the list of [availables scopes](https://help.akeneo.com).
+You can consult the list of [availables scopes](/apps/oauth2-scopes.html#supported-scopes).
```

Comment:
```suggestion
You can consult the list of [availables scopes](/apps/oauth2-scopes.html).
```

I think it's better to simply redirect to the full page.
If there is important information in the introduction, we should not skip it.